### PR TITLE
LIVE-3150 - LLM - Receive flow crash after merge issue

### PR DESCRIPTION
**Pull request by LFBarreto**
### 📝 Description

_Fixes issue introduced ater failed merge in LLM Receive Flow_

### ❓ Context

- **Impacted projects**: `ledger-live-mobile` 
- **Linked resource(s)**: [LIVE-3150] 

### ✅ Checklist

- [ ] **Test coverage** 
- [X] **Atomic delivery** 
- [X] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-3150]: https://ledgerhq.atlassian.net/browse/LIVE-3150?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hedi-edelbloute — APPROVED** [review]

File: apps/ledger-live-mobile/src/screens/ReceiveFunds/03a-ConnectDevice.tsx [hunk 1]
```diff
@@ -102,6 +102,7 @@ export default function ConnectDevice({ navigation, route }: Props) {
   }
 
   const mainAccount = getMainAccount(account, parentAccount);
+  const currency = mainAccount && getAccountCurrency(mainAccount)
```
Comment (Justkant) [inline apps/ledger-live-mobile/src/screens/ReceiveFunds/03a-ConnectDevice.tsx:105]: `getMainAccount` will throw if there is no `mainAccount` so the check is a bit redundant IMO

**Review by Justkant — CHANGES_REQUESTED** [review]
We can probably remove the check before `getAccountCurrency`

**Comment by Justkant** [comment]
We also need a changeset please

**Comment by LFBarreto** [comment]
> We also need a changeset please

I dont want any traces in changelogs of this

**Review by Justkant — APPROVED** [review]

**Pull request merged**